KlingonAssistant/data/mem-08-m.xml

```diff
@@ -4455,6 +4455,25 @@ When applied to {tlhIm:n}, {mutlh:v:nolink} means to assemble the carpet. The ve
       <column name="source">[1] {TKDA:src}</column>
     </table>
     <table name="mem">
+      <column name="_id">21655</column>
+      <column name="entry_name">muylIS</column>
+      <column name="part_of_speech">n</column>
+      <column name="definition">eyelash</column>
+      <column name="definition_de">Augenlid</column>
+      <column name="synonyms"></column>
+      <column name="antonyms"></column>
+      <column name="see_also">{mIn:n}</column>
+      <column name="notes"></column>
+      <!--Chris Lipscombe posted this in a message on Facebook, saying Okrand prived it during a game of Hockey Pockey -->
```

Comment:
nm found it: https://www.facebook.com/groups/LearnKlingon/permalink/1386341591474053/